wordpress/wp-content/plugins/cds-base/classes/Modules/Site/SiteSettings.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CDS\Modules\Site;
+
+use InvalidArgumentException;
```

Comment:
Does this get used?  It looks like I can enter any value I want by editing the "value" in the HTML.